pkg/controller/servicebindingrequest/reconciler_test.go

```diff
@@ -347,13 +347,15 @@ func TestEmptyApplicationSelector(t *testing.T) {
 
 	res, err := reconciler.Reconcile(reconcileRequest())
 	require.NoError(t, err)
-	require.True(t, res.Requeue)
+	require.False(t, res.Requeue)
 
 	namespacedName := types.NamespacedName{Namespace: reconcilerNs, Name: reconcilerName}
 	sbrOutput, err := reconciler.getServiceBindingRequest(namespacedName)
 	require.NoError(t, err)
 
 	require.Equal(t, conditions.BindingReady, sbrOutput.Status.Conditions[0].Type)
-	require.Equal(t, corev1.ConditionFalse, sbrOutput.Status.Conditions[0].Status)
+	// Currently the Conditions[0].Status would be true as application's absence won't cause error
+	// TODO New steps to conditions to be introduced - InjectionReady, CollectionReady
```

Comment:
Remind me please, do we have a tracked story/issue for this?

Reply:
https://github.com/redhat-developer/service-binding-operator/issues/381#issuecomment-606566580